### DISCUSSION TITLE
Tn5 insertion site adjustment

### DISCUSSION THREAD

**Discussion by z2e2** [GreenleafLab/ArchR#1530] [Questions / Documentation]
Hi ArchR team,

I am trying to use ArchR to process the `pbmc_granulocyte_sorted_10k_atac_fragments.tsv.gz` file downloaded from [10x Genomics](https://www.10xgenomics.com/resources/datasets/pbmc-from-a-healthy-donor-granulocytes-removed-through-cell-sorting-10-k-1-standard-2-0-0). In your documentation, you mentioned that you do the +4/-5 bp trick on the input data. However, it seems that cell-ranger has adjusted the insertion site according to their [doc](https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/algorithms/overview). There is no strand information (plus-stranded/minus-stranded fragments) in the fragments file from cell-ranger anyway.  I am wondering if ArchR only apply the +4/-5 bp adjustment on BAM input and doesn't adjust the insertion sites for fragments from cell-ranger output? 

Best,
Joe

**Comment by rcorces** [comment]
This is stated in the parameter definition for offset adjustment. Shifting only happens for BAM files

https://github.com/GreenleafLab/ArchR/blob/79953a93ae376a500e681aa3b5ba669af6bb58e1/R/CreateArrow.R#L47-L52
